sdk/healthinsights/azure-healthinsights-cancerprofiling/README.md

```diff
@@ -0,0 +1,263 @@
+# Azure Cognitive Services Health Insights Cancer Profiling client library for Python
+
+[Health Insights](https://review.learn.microsoft.com/azure/azure-health-insights/?branch=release-azure-health-insights) is an Azure Applied AI Service built with the Azure Cognitive Services Framework, that leverages multiple Cognitive Services, Healthcare API services and other Azure resources.
+
+The [Cancer Profiling model][cancer_profiling_docs] receives clinical records of oncology patients and outputs cancer staging, such as clinical stage TNM categories and pathologic stage TNM categories as well as tumor site, histology.
+
+
+
+## Getting started
+
+### Prerequisites
+
+- Python 3.7 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing Cognitive Services Health Insights instance.
+
+
+### Install the package
+
+```bash
+python -m pip install azure-healthinsights-cancerprofiling
```

Comment:
nit: you can just run `pip install azure-healthinsights-cancerprofiling`

Reply:
done